staging/src/k8s.io/pod-security-admission/policy/check_sysctls.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/pod-security-admission/api"
+)
+
+func init() {
+	addCheck(CheckSysctls)
+}
+
+// CheckSysctls returns a baseline level check
+// that limits the value of sysctls in 1.0+
+func CheckSysctls() Check {
+	return Check{
+		ID:    "sysctls",
+		Level: api.LevelBaseline,
+		Versions: []VersionedCheck{
+			{
+				MinimumVersion: api.MajorMinorVersion(1, 0),
+				CheckPod:       sysctls_1_0,
+			},
+		},
+	}
+}
+
+var (
+	sysctls_allowed_types_1_0 = sets.NewString(
+		"",
+		"kernel.shm_rmid_forced",
+		"net.ipv4.ip_local_port_range",
+		"net.ipv4.tcp_syncookies",
+		"net.ipv4.ping_group_range",
+	)
+)
+
+func sysctls_1_0(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) CheckResult {
+	var forbiddenDetails []string
+
+	checkSysctlsOptions := func(path *field.Path, sysctls []corev1.Sysctl) {
+		for _, sysctl := range sysctls {
+			if !sysctls_allowed_types_1_0.Has(sysctl.Name) {
+				forbiddenDetails = append(forbiddenDetails, path.Child("securityContext", "sysctl", "name").String())
```

Comment:
just the forbidden sysctl name is probably sufficient instead of a field path, since there's only one set of sysctls in the pod (they don't have to go hunting through various container definitions, etc)

Reply:
Done....
Hope i understand it right here.